front/app/api/idea_statuses/useIdeaStatuses.ts

```diff
@@ -4,15 +4,29 @@ import { CLErrors } from 'typings';
 import fetcher from 'utils/cl-react-query/fetcher';
 
 import ideaStatusesKeys from './keys';
-import { IIdeaStatuses, IdeaStatusesKeys } from './types';
+import {
+  IIdeaStatuses,
+  IdeaStatusesKeys,
+  IdeaStatusesQueryParams,
+} from './types';
 
-const fetchIdeaStatuses = () =>
-  fetcher<IIdeaStatuses>({ path: '/idea_statuses', action: 'get' });
+const fetchIdeaStatuses = ({ participation_method }: IdeaStatusesQueryParams) =>
+  fetcher<IIdeaStatuses>({
+    path: '/idea_statuses',
+    action: 'get',
+    queryParams: { participation_method },
+  });
 
-const useIdeaStatuses = ({ enabled = true } = {}) => {
+const useIdeaStatuses = ({
+  enabled = true,
+  participation_method,
+}: {
+  enabled?: boolean;
+  participation_method: IdeaStatusesQueryParams['participation_method'];
+}) => {
   return useQuery<IIdeaStatuses, CLErrors, IIdeaStatuses, IdeaStatusesKeys>({
-    queryKey: ideaStatusesKeys.lists(),
-    queryFn: fetchIdeaStatuses,
+    queryKey: ideaStatusesKeys.list({ participation_method }),
+    queryFn: () => fetchIdeaStatuses({ participation_method }),
     enabled,
```

Comment:
FYI - @sebastienhoorens Here's where the `enabled` value is used within `useQuery`